pyvo/dal/tests/test_sia.py

```diff
@@ -51,15 +53,3 @@ def test_search():
     _test_result(result)
 
 
-class TestSIAService:
```

Comment:
Why was this removed?

Reply:
Good question. By mistake I suppose. Added back.